src/systemctl/systemctl.c

```diff
@@ -7773,6 +7887,8 @@ static int systemctl_help(void) {
                "  daemon-reexec                       Reexecute systemd manager\n"
                "  log-level [LEVEL]                   Get/set logging threshold for manager\n"
                "  log-target [TARGET]                 Get/set logging target for manager\n"
+               "  service-log-level SERVICE [LEVEL]   Get/set logging threshold for service\n"
+               "  service-log-target SERVICE [TARGET] Get/set logging target for service\n"
```

Comment:
grmbl. yet another debugging verb in systemctl. I am not a fan of that. I wish we could at least fold this into the existing  "log-level" verb, like we do for "show" where you can specify a unit but if you don#t the call applies to the manager...

But the ambiuity kills this... i.e. having a service called "error.service" would mean "systemctl log-level error" would not have a clear meaning anymore...

Reply:
Why is this under "Manager State Commands" section? it's got nothing to do with the manager state, it's a per service thing